internal/lagoon/client/_lgraphql/environmentByName.graphql

```diff
@@ -1,19 +0,0 @@
-query (
-  $name: String!,
-  $project: Int!) {
-    environmentByName(
-      name: $name,
-      project: $project)
-      {
-        id
-        name
-        route
-        routes
-        deployType
-        environmentType
-        openshiftProjectName
-        updated
-        created
-        deleted
-      }
-  }
```

Comment:
Is this file accidentally deleted? It is used by `internal/lagoon/client/query.go` for the EnvironmentByName query, which is used in imports.